src/spi.rs

```diff
@@ -210,6 +239,17 @@ where
         }
     }
 
+    fn read_byte(&mut self) -> Result<u8, self::Error> {
```

Comment:
Since this is just a passthrough for `get_param` maybe we should only use one or the other?

Reply:
Thanks @calebbourg, a very good thought - I converged both down into a single `get_byte()` function.